### DISCUSSION TITLE
How to make backup and transfer it?

### DISCUSSION THREAD

**Discussion by Mozitux** [mediacms-io/mediacms#496] [General]
Hello;

I've a previous installation of MediaCMS on my old server and I wish to make a backup to transfer to my new one.
How can I proceed?

**Comment by mgogoulos** [comment]
Hi, there's some documentation on https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md but there could definitely be a backup section with more details. 

Is it a single server installation or a docker installation? 

For the single server installation, I'd say that you can do it following these steps:
a) create MediaCMS installation on new server (install system with setup.sh etc)
b) get a DB backup on the old server through pg_dump and add it on the new server with pg_restore as mediacms_new. Then you have to edit cms/settings.py or cms/local_settings.py and provide the new database name, eg mediacms_new
c) copy the whole media_files directory to the new server 
d) run migrations `python manage.py migrate` to get any schema changes that happened on the meanwhile (since you initially installed on the old server)

basically, this is it, give it a try and let us know how it goes!

**Reply by Mozitux** [reply]
Thanks, I will try!

**Reply by Mozitux** [reply]
@mgogoulos I try to make a backup with pgadmin4 but to connect to the mediacms db, pgadmin4 need a password, where can I found it?

**Reply by mgogoulos** [reply]
check file cms/settings.py (or cms/local_settings.py if you have created this file to override settings), you will find the password as part of the DATABASES dictionary
